src/api/apiFetch.js

```diff
@@ -42,6 +42,13 @@ export const apiCall = async (
     networkActivityStart(isSilent);
     const response = await apiFetch(auth, route, params);
 
+    if (response.status === 401) {
```

Comment:
We should not just change the natural way of handling errors - exceptions, with a custom old-school error code returning.